app/screens/settings/display_theme/display_theme.tsx

```diff
@@ -26,14 +26,14 @@ type DisplayThemeProps = {
 const DisplayTheme = ({allowedThemeKeys, componentId, currentTeamId, currentUserId}: DisplayThemeProps) => {
     const serverUrl = useServerUrl();
     const theme = useTheme();
-    const initialTheme = useMemo(() => theme, [/* dependency array should remain empty */]);
+    const initialTheme = useRef(theme);
```

Comment:
React best practice to use useRef vs useMemo to store value that persist throughout re-rendering, and also would not cause any re-rendering.